user-onboarding/src/Form.js

```diff
@@ -76,14 +76,16 @@ const FormikForm = withFormik({
         terms: Yup.boolean().oneOf([true])
     }),
 
-    handleSubmit(values, { setStatus }) {
+    handleSubmit(values, { setStatus, resetForm }) {
```

Comment:
all of the premade functions and methods do make formiks smoother than vanilla forms